### PR TITLE
Create a first faq concerning the tech stack.

### PR DESCRIPTION
**Pull request by fretchen**
Some of the questions concerning this little documentation always come back. So I just wanted to note some answers directly on the website...

### CODE REVIEW

File: docs/tech-faq.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+comments: true
+---
+
+# FAQ on the technology
+
+On this page, we answer some of the most frequent questions that people have concerning the website etc.
```
Comment (JottzumO) [inline docs/tech-faq.md:7]: how about: "..that people have concerning the website and the features of the website." Cutting out the "etc" in this case?

File: docs/tech-faq.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Question: I heard that this website is on the github. What does this mean ?
```
Comment (JottzumO) [inline docs/tech-faq.md:9]: GitHub

**Review by JottzumO — COMMENTED** [review]
Maybe add another question such as: "What if I would like to contribute/ How can I contribute if I have important information/ an update on something?"

File: docs/tech-faq.md (hunk 2 above)
Comment (fretchen) [reply docs/tech-faq.md]: Updated in the new commit.

**Review by JottzumO — APPROVED** [review]

**Pull request merged**